spec/features/upload_download_mous_spec.rb

```diff
@@ -23,14 +23,23 @@
     end
 
     context 'uploading the signed mou' do
-      it 'can upload and download a version of the mou' do
+      before do
         visit mou_index_path
-
         attach_file("signed_mou", Rails.root + "spec/fixtures/mou.pdf")
         click_on 'Upload'
+      end
 
+      it 'can upload and download a version of the mou' do
         expect(page).to have_content("MOU uploaded successfully.")
       end
+
+      it 'allows the user to see when the mou was uploaded' do
+        expect(page).to have_content("A signed MoU was uploaded on")
```

Comment:
I know it's not good to have multiple assertions per it block but we're executing the before block 3 times here to check the text on the same page.

What are your thoughts?

Reply:
Condense it into one IT block sounds good